PropertySuggester.php

```diff
@@ -61,5 +61,10 @@
 	31 // instance of
 );
 
+global $wgPropertySuggesterInitialSuggestions;
+$wgPropertySuggesterInitialSuggestions = array(
+	31, // instance of
+	279 // subclass of
```

Comment:
I suggest to keep this base configuration as simple as possible and remove 279 here. Add it to the actual configuration on the Wikimedia cluster.